lib/util/glob-util.js

```diff
@@ -88,6 +89,18 @@ function resolveFileGlobPatterns(patterns, options) {
     return patterns.filter(p => p.length).map(processPathExtensions);
 }
 
+const dotfilesPattern = /(?:(?:^\.)|(?:[/\\]\.))[^/\\.].*/;
+
+const memoGetIgnorePaths = _.memoize(
```

Comment:
I'm not convinced this function (or the one below) needs a "memo" prefix, but I won't insist on a change here.